tests/scale/virt-api.go

```diff
@@ -74,7 +74,7 @@ var _ = Describe("[sig-compute] virt-api scaling", decorators.SigCompute, func()
 		return expectedReplicas
 	}
 
-	It("virt-api replicas should be scaled as expected", func() {
+	It("virt-api replicas should be scaled as expected", decorators.Conformance, func() {
```

Comment:
This depends on the env, I think it does not meet criteria

Reply:
Oh, that's right! Nice catch!